samples/csharp/MicrosoftEduGraphSamples/Program.cs

```diff
@@ -14,5 +14,5 @@
 
 // Create an instance of the class you want to test and call the desired workflow method.
 // Each flow represents a common entire process that can be tested using Microsoft Graph SDK, all the sample flows are located in the "workflows" folder.
-AssignmentWorkflow assignmentWorkflow = new AssignmentWorkflow(config);
-await assignmentWorkflow.GetMeAssignmentsFromNonArchivedClassesAsync();
\ No newline at end of file
+ModuleWorkflow assignmentWorkflow = new ModuleWorkflow(config);
+await assignmentWorkflow.ClassworkAsync();
```

Comment:
you can revert this change, it was for testing purposes only

Reply:
reverted it, Resolved